frontend/add-client/services.component.tsx

```diff
@@ -2,11 +2,23 @@ import React, { useState } from "react";
 import { StepComponentProps, Step } from "./add-client.component";
 import agendaIconUrl from "../../icons/148705-essential-collection/svg/agenda.svg";
 import { useCss } from "kremling";
+import easyFetch from "../util/easy-fetch";
 
 export default function Services(props: StepComponentProps) {
-  const [services, setServices] = useState(defaultServices);
+  const [services, setServices] = useState([]);
+  const [checkedServices, setCheckedServices] = useState([]);
   const scope = useCss(css);
 
+  React.useEffect(() => {
```

Comment:
So this is what a way to use useEffect... What does the AbortController component do? I am assuming is a react component?

Reply:
The abort controller allows us to cancel the network request and not call setServices if the user navigates to a different page while we're waiting on the network request.

See https://stackoverflow.com/questions/31061838/how-do-i-cancel-an-http-fetch-request